src/main/res/values/strings.xml

```diff
@@ -467,4 +467,5 @@ Please report any problems you encounter via Github.</string>
     <!-- Format string for folder size, eg "500 MiB / 1 GiB" -->
     <string name="folder_size_format">%1$s / %2$s</string>
 
+    <string name="appconfig_receiver_background_enabled">Stopping SyncThingService is not supported when running in background enabled.</string>
```

Comment:
As I said, it's Syncthing without capital T :p 

And I would just write "Stopping Syncthing is not supported", no need to confuse people what a "service" is.